src/Apps/Collect/FilterState.tsx

```diff
@@ -38,7 +40,7 @@ export class FilterState extends Container<State> {
 
   constructor(props: State) {
     super()
-    this.tracking = props.tracking
+    this.tracking = props && props.tracking
```

Comment:
```js
if (props.tracking) {
  this.tracking = 
}
```

Which also needs to be guarded here: 

https://github.com/artsy/reaction/blob/b11e7a7f1662bd86c6488f72986c0539b3bcafa7/src/Apps/Collect/FilterState.tsx#L148

Reply:
(Or removed for now, since this pattern should likely be revised and hoisted out of state management)